admin/sections/config/form.tsx

```diff
@@ -48,7 +50,10 @@ export const ConfigForm = ({
         setConfigured(false);
       }
       // Reload the chat iframe to apply the changes from the new config
-      demoChatIframeRef.current?.reloadIframe();
+      const changedFields = form.formState.dirtyFields;
+      if (changedFields.conversation_starters) {
+        demoChatIframeRef.current?.reloadIframe();
+      }
```

Comment:
TODO: can be another PR (show reload button if config was changed and do reload only explicitly if the user clicks the reload button)

Reply:
i just added a reload button on top of the iframe, it will always show in the top right. So i think we no longer need auto reload iframe from now.